app/models/bulk_upload.rb

```diff
@@ -78,37 +78,18 @@ def unpend
     end
   end
 
+  SHARED_VALUE_CHECKS = %w[retirement_value_check].freeze
```

Comment:
this refactoring is a) nicer and b) allows testing that the value checks are all being confirmed in bulk_upload_spec (previously some of the value_check fields were missing from unpend_and_confirm_soft_validations)

Reply:
Hmm, I wonder if getting it from `log.form.questions.filter_the_questions_by_interruption_screen.map(&:id).uniq` would help us avoid missing some here again?

Reply:
yeah okay, I'll move the explicit checking to the test so we've still got it somewhere, nice point